modules/installation-configuration-parameters.adoc

```diff
@@ -1833,11 +1833,11 @@ a|Multiple IP addresses
 |String
 
 |`platform.vsphere.failureDomains.region`
-|You define a region by using a tag from the `openshift-region` tag category. The tag must be attached to the vCenter datacenter.
```

Comment:
Feedback Questions:

* _Q1: Is it ok if we set dummy values to those parameters when we deploy OpenShift with single region/zone?
* Q2: The manual[2] says "The tag must be attached to the vCenter datacenter". But really do we have to create openshift-region/zone tags on vCenter datacenter even when we deploy OpenShift with single region/zone?_

My assumption was that the user did not need to do anything as default values were automatically specified if a user wants to run a cluster in a single vSphere datacenter?

Reply:
> My assumption was that the user did not need to do anything as default values were automatically specified if a user wants to run a cluster in a single vSphere datacenter?

Nope, they need to define values of zone and region. 

> Does a user need to attach tags to a single vSphere datacenter environment?

no

> A dummy value should never be a thing? A user will always have more accurate info that they can specify as values for the targeted parameters?

The only time when there are generated values is when the depreciated platform spec is used.